dj_rql/filter_cls.py

```diff
@@ -890,7 +890,11 @@ def _convert_value(cls, django_field, str_value, use_repr=False):
             return float(val)
 
         elif filter_type == FilterTypes.DECIMAL:
-            return round(float(val), django_field.decimal_places)
+            integer_part = len(val.split('.', 1)[0])
```

Comment:
What will be integer_part for negative (-1) value? Should be 1, not 2.

Reply:
Done, thanks.